.github/workflows/test_definitions.yml

```diff
@@ -122,4 +122,4 @@ jobs:
     - name: Dispatch Docs Publish workflow
       # https://github.com/orgs/community/discussions/26323#discussioncomment-3251448
       run: |
-        curl -XPOST -u "${{ secrets.PAT_USERNAME}}:${{secrets.PAT_TOKEN}}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/UBCSailbot/docs/actions/workflows/publish.yml/dispatches --data '{"ref": "main"}'
+        curl -XPOST -u "${{ secrets.PAT_USERNAME}}:${{secrets.PAT_TOKEN}}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/UBCSailbot/docs/actions/workflows/publish.yml/dispatches --data '{"ref": "main"}'
```

Comment:
lol why do you always remove the empty line at the end of the file? You can check the files changes tab to check that no unintended changes were made

Reply:
Why does it keep doing that? I'm going to check